.travis.yml

```diff
@@ -8,8 +8,8 @@ git:
 branches:
   only:
     - develop
+    - release/4.2.x
     - "/^epic\\/.*$/"
-    - "/^release\\/.*$/"
```

Comment:
Why would we only want to run the CI on 4.2.x and not any other release branches?

Reply:
it's on the PR description :)